inc/msp/msp_msg_maker.hpp

```diff
@@ -0,0 +1,616 @@
+#ifndef MSP_MSG_MAKER_HPP
+#define MSP_MSG_MAKER_HPP
+#include "msp_msg.hpp"
+
+namespace msp {
+
+class message_maker {
+public:
+    message_maker(FirmwareVariant v = FirmwareVariant::INAV) : fw_variant(v) {};
+
+    void set_default_variant(FirmwareVariant v) {
+        fw_variant = v;
+    }
+    
+    
+    std::shared_ptr<Message> operator() (ID& id, FirmwareVariant& v)
+    {
+        std::shared_ptr<Message> msg_ptr;
+        switch(id) {
```

Comment:
So this is like a factory, where you create a message instance from a given ID? Having it here in this file increases the maintenance costs when new messages are added.
Can you think of a way using template metaprogramming or preprocessor macros in such a way that this could be defined within the message definition?

Reply:
I forgot I committed this file... It's actually not used in the project.

Reply:
Ok, then just remove it.